resources/ts/Components/IntegrationCard.tsx

```diff
@@ -29,6 +29,18 @@ export const IntegrationCard = ({ id, name, type, status, onEdit }: Props) => {
     }, 1000);
   }
 
+  const getProductUrl = () => {
+    if (type === "entry-api") {
+      return "/uitdatabank/entry-api%2Fintroduction";
+    }
+    if (type === "search-api") {
+      return "/uitdatabank/search-api%2Fintroduction";
+    }
+    if (type === "widgets") {
+      return "/widgets/aan-de-slag";
+    }
+  };
+
```

Comment:
You could also use a object as a map outside the component:

```ts
const productTypeToPath = {
    "entry-api": encodeURIComponent("/uitdatabank/entry-api/introduction"),
    "search-api": encodeURIComponent("/uitdatabank/search-api/introduction"),
    "widgets": encodeURIComponent("/widgets/aan-de-slag")
}
```

And then use it like:
```ts
<Link
    href={t("integrations.documentation.action_url", {
        product: productTypeToPath[type],
    })}
 >
```

I use `encodeURIComponent` here to make the url more readable in code but also still encoding it